terraform/aws/modules/cluster-post-installation/prometheus-client.tf

```diff
@@ -0,0 +1,13 @@
+resource "helm_release" "prometheus-client" {
+  name       = "mattermost-cm-prometheus-client"
+  namespace  = "monitoring"
+  repository = "${data.helm_repository.stable.metadata.0.name}"
+  chart      = "stable/prometheus"
+  wait       = false
+  values = [
+    "${file("../../../../../chart-values/prometheus-client_values.yaml")}"
```

Comment:
I'm not incredibly familiar with terraform but this looks a little fragile where if we move directories around a little bit it's going to break. Is there a cleaner way we can reference this file?

Reply:
@jwilander Unfortunately the only way to get the data is by specifying a file and the file needs a path. Because we have lots of charts needed I chose to put them all under a specific chart-values directory, which sits in the home directory of the repo. If we need to change the directories for any reason we will need to update the paths.